core/src/io/anuke/mindustry/input/Binding.java

```diff
@@ -19,6 +19,16 @@
     dash(KeyCode.SHIFT_LEFT),
     gridMode(KeyCode.BACKTICK),
     gridModeShift(KeyCode.ALT_LEFT),
+    build_category_1(KeyCode.NUM_1),
```

Comment:
All of these need to be put in `bundle.properties` to show up properly.

Reply:
I've added it to the base bundles.properties, does it also need to be added to the other region-specific translations or do they fall back to english until properly translated?

Reply:
It falls back, and there's a script to add them to other translations that I run occasionally.